### PR TITLE
Add in more references for css-color tests

### PR DESCRIPTION
**Pull request by csnardi**
These should be the last tests missing references in css-color.

The tests `t32-opacity-basic-0.6-a.xht`, `t422-rgba-a0.6-a.xht`, and `t425-hsla-basic-a.xht` are failing on all platforms with Travis, but the references are basically verifying the reference `<p>` in the test itself, so I'm not sure if this is just something that all implementations have done incorrectly or what.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/21897)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/317897142)

**Comment by gsnedders** [comment]
> The tests `t32-opacity-basic-0.6-a.xht`, `t422-rgba-a0.6-a.xht`, and `t425-hsla-basic-a.xht` are failing on all platforms with Travis, but the references are basically verifying the reference `<p>` in the test itself, so I'm not sure if this is just something that all implementations have done incorrectly or what.

I'm not sure we define the alpha blend mode well enough to actually assert that they exactly match, hence we may just be very slightly out. (I haven't actually looked at the screenshots here, but I expect we're just slightly out colour-wise.) @svgeesus?

**Review by svgeesus — COMMENTED** [review]
Sorry to have missed this PR. Will look into this in detail. Two aspects to consider:
- numerically accurate result and error limits
- actual spec text backing up exactly what the test is checking

File: css/css-color/t421-rgb-hex3-expand-b-ref.html
Comment (svgeesus) [inline css/css-color/t421-rgb-hex3-expand-b-ref.html]: Those three changes are clearly correct, astonished the earlier values were in the test to be honest

File: css/css-color/t422-rgba-a0.6-a.xht
Comment (svgeesus) [inline css/css-color/t422-rgba-a0.6-a.xht]: yes

File: css/css-color/t425-hsla-basic-a.xht
Comment (svgeesus) [inline css/css-color/t425-hsla-basic-a.xht]: No, leave the bold in there. It is easier to see the correct color on bolder text.

File: css/css-color/t32-opacity-zorder-c-ref.html
Comment (svgeesus) [inline css/css-color/t32-opacity-zorder-c-ref.html]: Not super happy on 'very-nearly-green' but ok. 
Suggest adding explicit  "and no red"

File: css/css-color/t32-opacity-basic-0.6-a-ref.html
Comment (svgeesus) [inline css/css-color/t32-opacity-basic-0.6-a-ref.html]: Yes, clearly

**Review by svgeesus — CHANGES_REQUESTED** [review]

**Comment by svgeesus** [comment]
Still waiting for small changes requested in review.

**Comment by svgeesus** [comment]
@csnardi waiting for your changes based on review

**Comment by svgeesus** [comment]
> @csnardi waiting for your changes based on review

Small changes were requested a while ago. Should be easy.

**Comment by svgeesus** [comment]
Looks like the requested changes are not going to be made. Maybe someone else should take over the changes?

**Comment by svgeesus** [comment]
OK, given the original poster seems disinterested in making any suggested changes, I'm going to change my review to accept and then work on improvements on top of that. Not ideal, but the only way to unblock.

**Review by svgeesus — APPROVED** [review]

**Comment by gsnedders** [comment]
> Not ideal, but the only way to unblock.

You could've just created a new PR that's a superset of this!

**Pull request merged**
